api.planx.uk/modules/analytics/metabase/collection/getTeamIdAndMetabaseId.ts

```diff
@@ -0,0 +1,38 @@
+import { gql } from "graphql-request";
+import { $api } from "../../../../client/index.js";
+
+interface GetMetabaseId {
+  teams: {
+    id: number;
+    name: string;
+    metabaseId: number | null;
+  }[];
+}
+
+export const getTeamIdAndMetabaseId = async (name: string) => {
+  try {
+    const response = await $api.client.request<GetMetabaseId>(
+      gql`
+        query GetTeamAndMetabaseId($name: String!) {
+          teams(where: { name: { _ilike: $name } }) {
```

Comment:
`_ilike` _could_ match both "North Somerset" and "South Somerset" - it would be better to match using `_eq` on team slugs if possible which are guaranteed to be unique.

Reply:
Great catch! Updated.